docs/tools/vdb_table/data/vectara.json

```diff
@@ -57,9 +57,9 @@
     "comment": ""
   },
   "bm25": {
-    "support": "none",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.vectara.com/docs/learn/hybrid-search",
```

Comment:
it'd be best if you actually mention BM25 somewhere in this doc page, or in the API reference, so that people migrating onto Vectara can know for sure what lexical_interpolation signifies.

Can you say which formula/flavor of BM25 has been used?

Reply:
Thanks. great feedback and I've asked our docs team to add that.
I believe it is currently the original BM25.

Reply:
Okay - it's added to the docs, see for example: https://docs.vectara.com/docs/learn/hybrid-search#enable-hybrid-search